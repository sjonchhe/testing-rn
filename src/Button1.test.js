import React from 'react';
import 'react-native';
import Button1 from './Button1';
import renderer from 'react-test-renderer';

test('Button1 Snapshot', () => {
  const snap = renderer.create(
    <Button1 title="there" />
  ).toJSON();
  expect(snap).toMatchSnapshot();
})