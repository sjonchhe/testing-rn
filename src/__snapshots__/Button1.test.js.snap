// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button1 Snapshot 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignSelf": "stretch",
      "backgroundColor": "#38ba7d",
      "borderBottomColor": "#1e6343",
      "borderBottomWidth": 6,
      "borderWidth": 1,
      "height": 45,
      "justifyContent": "center",
      "marginLeft": 15,
      "marginRight": 15,
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "color": "#fff",
        "fontSize": 16,
        "fontWeight": "600",
      }
    }
  />
</View>
`;
